test/integration/networks/networks_test.go

```diff
@@ -350,24 +349,16 @@ func TestNetworks(t *testing.T) {
 					// networks.DefaultVerify(assert)
 
 					servicePerimeterLink := fmt.Sprintf("accessPolicies/%s/servicePerimeters/%s", policyID, networks.GetStringOutput("restricted_service_perimeter_name"))
-					accessLevel := fmt.Sprintf("accessPolicies/%s/accessLevels/%s", policyID, networks.GetStringOutput("restricted_access_level_name"))
+					accessLevel := fmt.Sprintf("accessPolicies/%s/accessLevels/%s", policyID, networks.GetStringOutput("access_level_name"))
 					networkNames := getNetworkResourceNames(envCode, networkMode, firewallMode)
 
-					servicePerimeter := gcloud.Runf(t, "access-context-manager perimeters describe %s --policy %s", servicePerimeterLink, policyID)
-					assert.Equal(servicePerimeterLink, servicePerimeter.Get("name").String(), fmt.Sprintf("service perimeter %s should exist", servicePerimeterLink))
-					listLevels := utils.GetResultStrSlice(servicePerimeter.Get("status.accessLevels").Array())
-					assert.Contains(listLevels, accessLevel, fmt.Sprintf("service perimeter %s should have access level %s", servicePerimeterLink, accessLevel))
-					listServices := utils.GetResultStrSlice(servicePerimeter.Get("status.restrictedServices").Array())
-					assert.Subset(listServices, restrictedServices, fmt.Sprintf("service perimeter %s should restrict all supported services", servicePerimeterLink))
-
-					listIngressPolicies := servicePerimeter.Get("status.ingressPolicies").Array()
-					assert.Equal(len(listIngressPolicies), len(ingressPolicies), fmt.Sprintf("service perimeter %s should have the same number of input ingress policies (%d)", servicePerimeterLink, len(ingressPolicies)))
-					ingressOp := servicePerimeter.Get("status.ingressPolicies.0.ingressTo.operations.0")
-					assert.Equal(operationService, ingressOp.Get("serviceName").String(), fmt.Sprintf("ingress policy should support operations on %s", operationService))
-					listEgressPolicies := servicePerimeter.Get("status.egressPolicies").Array()
-					assert.Equal(len(listEgressPolicies), len(egressPolicies), fmt.Sprintf("service perimeter %s should have the same number of input egress policies (%d)", servicePerimeterLink, len(egressPolicies)))
-					egressOp := servicePerimeter.Get("status.egressPolicies.0.egressTo.operations.0")
-					assert.Equal(operationService, egressOp.Get("serviceName").String(), fmt.Sprintf("egress policy should support operations on %s", operationService))
+					servicePerimeter, err := gcloud.RunCmdE(t, fmt.Sprintf("access-context-manager perimeters dry-run describe %s --policy %s", servicePerimeterLink, policyID))
+					assert.NoError(err)
+					assert.True(strings.Contains(servicePerimeter, servicePerimeterLink), fmt.Sprintf("service perimeter %s should exist", servicePerimeterLink))
```

Comment:
Looks 357 & 358 are still causing problems, due to the different format of describing a dryrun perimeter.. 

I'm getting an error like
>Step #17 - "verify-networks":     networks_test.go:357: 
Step #17 - "verify-networks":         	Error Trace:	/workspace/test/integration/networks/networks_test.go:357
Step #17 - "verify-networks":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.15.1/pkg/tft/terraform.go:605
Step #17 - "verify-networks":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.15.1/pkg/tft/terraform.go:637
Step #17 - "verify-networks":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.15.1/pkg/utils/stages.go:31
Step #17 - "verify-networks":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.15.1/pkg/tft/terraform.go:637
Step #17 - "verify-networks":         	Error:      	Should be true
Step #17 - "verify-networks":         	Test:       	TestNetworks/production
Step #17 - "verify-networks":         	Messages:   	service perimeter accessPolicies/886321771265/servicePerimeters/sp_p_shared_restricted_default_perimeter_fb3a should exist
Step #17 - "verify-networks":     networks_test.go:358: 
Step #17 - "verify-networks":         	Error Trace:	/workspace/test/integration/networks/networks_test.go:358
Step #17 - "verify-networks":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.15.1/pkg/tft/terraform.go:605
Step #17 - "verify-networks":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.15.1/pkg/tft/terraform.go:637
Step #17 - "verify-networks":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.15.1/pkg/utils/stages.go:31
Step #17 - "verify-networks":         	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.15.1/pkg/tft/terraform.go:637
Step #17 - "verify-networks":         	Error:      	Should be true
Step #17 - "verify-networks":         	Test:       	TestNetworks/production
Step #17 - "verify-networks":         	Messages:   	service perimeter accessPolicies/886321771265/servicePerimeters/sp_p_shared_restricted_default_perimeter_fb3a should have access level accessPolicies/886321771265/accessLevels/alp_p_shared_restricted_members_fb3a

Reply:
Looking at it